### ISSUE TITLE
Unable to find ExportDeclaration

### ISSUE THREAD

**Issue by thejameskyle** [fkling/esprima_ast_explorer#43]
http://astexplorer.net/#/0yIbDQg5KK

Input:
```js
export default foo;
```

Transform:
```js
j(file.source).find(j.ExportDeclaration).remove().toSource();
```

Output:
```js
export default foo;
```

**Comment by thejameskyle** [comment]
Oh weird, it's `ExportDefaultDeclaration` but the AST viewer shows `ExportDeclaration`

**Comment by fkling** [comment]
if you have never used the recast parser before, you have to configure it to use `babel` internally.

<img width="793" alt="screen shot 2015-11-03 at 10 38 26 am" src="https://cloud.githubusercontent.com/assets/179026/10917717/0903b40a-8217-11e5-852c-e41b963f91dd.png">

**Comment by fkling** [comment]
I didn't get around yet to let tools specify the parser settings.

**Comment by fkling** [comment]
I created #44.

**Comment by thejameskyle** [comment]
That modal is broken in Safari btw

![screen shot 2015-11-03 at 10 41 00 am](https://cloud.githubusercontent.com/assets/952783/10917788/67b0a7e2-8217-11e5-9751-f476b5e8cbcc.png)

**Comment by fkling** [comment]
Fixed setting dialog in Safari: 3c1361f6f144f5d70a995814d402fecf425c50d2.
